.ci/system-tests/run-system-tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Script to run the System Tests for HIRS
+
+set -e
+
+# Start System Testing Docker Environment
+cd .ci/docker
+
+docker-compose up -d
+
+tpm2_container_id="$(docker ps -aqf "name=hirs-aca-provisioner-tpm2")"
+echo "TPM2 Container ID: $tpm2_container_id"
+
+tpm2_container_status="$(docker inspect $tpm2_container_id --format='{{.State.Status}}')"
+echo "TPM2 Container Status: $tpm2_container_status"
+
+while [ $tpm2_container_status == "running" ] 
+do
+  sleep 5 
+
+  tpm2_container_status="$(docker inspect $tpm2_container_id --format='{{.State.Status}}')"
```

Comment:
For this loop we may wish to consider either throwing in an `echo` statement regarding the status of "still running". Alternatively, we could look at https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received. Although that would effectively just throw in something similar to the echo statement I mentioned above, but is even less descriptive.

My main concern here is that as we add more to these systems tests if any of them take longer than 10 minutes to run without printing out any status, then Travis will terminate the job.